### PR TITLE
Switch to Flask-WTF in a few places

### PR DESCRIPTION
**Pull request by mutantmonkey**
I haven't gotten around to some of the other forms yet, but I figured I would go ahead and submit this.

Flask-WTF's CSRF extension works as a drop-in replacement for SeaSurf, so we can get rid of that.

The Trackman pre-auth forms (register and reactivate) now use Flask-WTF and have better validation so that DJs won't trigger 500 errors if they try to enter information that is too long.

### CODE REVIEW

**Comment by echarlie** [comment]
On some quick testing: I don't know how often it will be an issue, but root@localhost is no longer a valid email to trackman

File: wuvt/trackman/forms.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+def strip_field(val):
+    if isinstance(val, basestring):
+        return val.strip()
+    else:
+        return val
+
+
+def artist_validate(form, field):
+    if field.data in current_app.config['TRACKMAN_ARTIST_PROHIBITED']:
+        raise ValidationError("That artist may not be entered.")
```
Comment (echarlie) [inline wuvt/trackman/forms.py]: do we actually have this field configured in production? Do we still need it?

Comment (mutantmonkey) [inline wuvt/trackman/forms.py]: Yes and historical Trackman data indicates it is necessary.

**Comment by mutantmonkey** [comment]
The Stripe version bump is minor enough that I figured I'd cover it in this.

I'm not sure what kind of explanation you want regarding the error handling...

**Pull request merged**